nautobot/core/testing/views.py

```diff
@@ -1015,9 +1016,16 @@ class BulkEditObjectsViewTestCase(ModelViewTestCase):
 
         bulk_edit_data = {}
 
-        def validate_object_data_after_bulk_edit(self, pk_list):
```

Comment:
Might be a breaking change to remove this method - may need to leave it around for now?